### PR TITLE
Updated process for running demos and OpenShift installation

### PR DESCRIPTION
**Pull request by mrsabath**

### CODE REVIEW

File: charts/ti-key-release-1/templates/configmap/tsi-mutate-configmap.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ data:
           value: {{ .Values.secrets.refreshSec }}
         volumeMounts:
           - name: tsi-secrets
-            mountPath: /usr/share/secrets
+            mountPath: /usr/share/secrets/tsi-secrets
```
Comment (lumjjb) [inline charts/ti-key-release-1/templates/configmap/tsi-mutate-configmap.yaml:27]: does the consumer of this have to be changed somewhere?

Comment (mrsabath) [reply charts/ti-key-release-1/templates/configmap/tsi-mutate-configmap.yaml:27]: Since we always enforce the local path for secrets in the container must start with `tsi-secrets`, this name would always be appended to the path in the sidecar, so it would be always properly processed by  `components/jwt-sidecar/get-vault-secrets.sh` script on the sidecar, then mounted to the container.

File: demos/demo.install-TSI.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# this script requires https://github.com/duglin/tools/tree/master/demoscript
+declare DEMOFILE=~/workspace/tools/demoscript/demoscript
+if [ ! -f "$DEMOFILE" ]; then
+    echo "$DEMOFILE does not exist."
+		exit 1
```
Comment (lumjjb) [inline demos/demo.install-TSI.sh:7]: nit: indent

File: demos/demo.install-TSI.sh [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+fi
+source ${DEMOFILE}
+
+SCRIPT_PATH=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+TSI_VERSION=$(cat ${SCRIPT_PATH}/../tsi-version.txt)
+#UTILS="${SCRIPT_PATH}/../utils"
+UTILS="../utils"
```
Comment (lumjjb) [inline demos/demo.install-TSI.sh:14]: hmm, why did you remove script path for this? What's the exception for utils?

File: demos/demo.install-TSI.sh [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+comment "Remove the TSI setup deployment..."
+doit "helm ls"
+doit "helm delete --purge tsi-setup"
+doit "$kk get po"
+
+exit 0
+
+
+
```
Comment (lumjjb) [inline demos/demo.install-TSI.sh:83]: These below are not called? can we comment maybe?

File: demos/demo.revoke-keys.pod.sh [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# this script requires https://github.com/duglin/tools/tree/master/demoscript
+declare DEMOFILE=~/workspace/tools/demoscript/demoscript
+if [ ! -f "$DEMOFILE" ]; then
+    echo "$DEMOFILE does not exist."
+		exit 1
```
Comment (lumjjb) [inline demos/demo.revoke-keys.pod.sh:6]: idnent

File: demos/demo.revoke-keys.pod.sh [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+doit kubectl -n test get po
+doit kubectl -n test get po
+#doit kubectl -n test exec -it $(kubectl -n test get po | grep myubuntu | awk '{print $1}' |  sed -n 1p ) -c myubuntu bash
+#doit watch ls -l /tsi-secrets/mysecret/
+#exit
+
+
+ttyDoit kubectl -n test exec -it $(kubectl -n test get po | grep myubuntu | grep Running | awk '{print $1}' |  sed -n 1p ) -c myubuntu bash 10<<EOF
+  cat /tsi-secrets/mysecret1
+	watch ls -l /tsi-secrets/
```
Comment (lumjjb) [inline demos/demo.revoke-keys.pod.sh:24]: indent

File: examples/vault/demo.vault-setup.manual.sh [hunk 7]
```diff
@@ -0,0 +1,133 @@
+
```
Comment (lumjjb) [inline examples/vault/demo.vault-setup.manual.sh:1]: shebang has to be on the first line i think? not sure what is the behavior otherwise
